frontend/src/app/plugins/shape.cljs

```diff
@@ -1377,7 +1399,16 @@
                        (u/display-not-valid :verticalSizing "Plugin doesn't have 'content:write' permission")
 
                        :else
-                       (st/emit! (dwsl/update-layout #{id} {:layout-item-v-sizing value})))))})))
+                       (st/emit! (dwsl/update-layout #{id} {:layout-item-v-sizing value})))))}
+
+
+                {:name "variants"
+                 :enumerable false
+                 :get
+                 (fn [self]
```

Comment:
The properties that have only `get` should be defined as `readonly` in the plugin-types.